lua_configuration/lua.go

```diff
@@ -0,0 +1,225 @@
+package main
```

Comment:
is this source file a UT? can we run it when building the code?

Reply:
This file is more like a plugin to facilitate the development of Lua scripts.